scripts/check-snarky-submodule.sh

```diff
@@ -5,7 +5,10 @@ set -eu
 cd src/lib/snarky
 
 CURR=$(git rev-parse HEAD)
+# temporarily skip SSL verification (for CI)
+git config http.sslVerify false
```

Comment:
Seems like this temporary workaround is ok but we could totally redeploy the cluster so that SSH (vs. HTTPS) can be used and avoid having to `fetch` without SSL.

Reply:
I'll make a new issue to track support SSL certs in the cluster

Reply:
Tracking https://github.com/CodaProtocol/coda/issues/4980